src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnTask.kt

```diff
@@ -0,0 +1,45 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.ef.iverksett.featuretoggle.FeatureToggleService
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Service
+@TaskStepBeskrivelse(
+        taskStepType = OpprettOppgaveForBarnTask.TYPE,
+        beskrivelse = "Oppretter oppgave for barn som fyller 1/2 eller 1 år",
+
+        )
+class OpprettOppgaveForBarnTask(val taskRepository: TaskRepository,
+                                val opprettOppgaveForBarnService: OpprettOppgaveForBarnService,
+                                val featureToggleService: FeatureToggleService) : AsyncTaskStep {
+
+    val DATE_FORMAT_ISO_YEAR_MONTH_DAY = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+    val INNEN_ANTALL_UKER = 1L
+
+    override fun doTask(task: Task) {
+        if (featureToggleService.isEnabled("familie.ef.iverksett.opprett-oppgaver-barnsomfylleraar")) {
+            opprettOppgaveForBarnService.opprettOppgaverForAlleBarnSomFyller(INNEN_ANTALL_UKER)
```

Comment:
For at ingen skal "forsvinne" mellom ukene våre tror jeg kanskje ikke vi kan bruke LocalDate.now() gjennom dato-sjekkene våre, men bør sende med et "sjekk-tidspunkt" fra tasken. 

Bør det lagres et "sjekktidspunkt" i metadata i tasken som vi bruker, eller holder det med localdate.now fra taskens kjøretid? Hvis noe feiler og vi må rekjøre dagen etter så frykter jeg at vi kanskje "mister" noen barn på veien - så heller mot at vi burde lagre triggerDag som metadata på tasken.

Reply:
Hva med en "referansedato", sånn at hvis man kjører "for sent", så vil man ta utgangspunkt i den første dagen etter den siste dagen som det ble sjekket for sist gang, og hvis man sjekker for tidlig, så tar man utgangspunkt i at fra og med dato er dagens dato ?